modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -0,0 +1,58 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="installing-pipelines-as-code-cli_{context}"]
+= Installing Pipelines as Code CLI 
+
+[role="_abstract"]
+In addition to installing Pipelines as Code on a {product-title} cluster, cluster administrators can install the `tkn-pac` CLI tool on local machines or as containers for testing.
+
+[discrete]
+.Procedure
+
+* Install the latest stable release on any system where the `oc` or `kubectl` command line tool is available:
```

Comment:
you can remove the installation part since osp 1.7 operator install pac by default with operator, i believe there is TektonConfig trickery to being able to disable it which we may want to document, 

cc @sm43

Reply:
with osp 1.7 pac is installed by default  
if user want to disable it then there is a field in `TektonConfig`
```
spec:
    addon:
      enablePipelinesAsCode: true
```